docs/debug/index.md

```diff
@@ -97,21 +97,86 @@ iteration    number    sum_so_far    list_so_far
 6            6         15            [0, 1, 2, 3, 4, 5]
 ```
 
+### Example 3: Multiple sequential loops
+
+The `AccumulationTable` context manager can track multiple sequential loops within the same `with` block. Each loop will generate its own separate table in the output.
+
+```python
+# demo.py
+from python_ta.debug import AccumulationTable
+
+
+def process_with_multiple_loops(numbers: list) -> int:
+    """Process numbers through multiple sequential loops.
+    """
+    sum_so_far = 0
+    i = 0
+    with AccumulationTable(["sum_so_far", "i"]):
+        for number in numbers:
+            sum_so_far = sum_so_far + number
+        for j in range(len(numbers)):
+            sum_so_far += 1
+        while i < 5:
+            i += 1
+            sum_so_far -= i
+
+    return sum_so_far
+
+
+if __name__ == '__main__':
+    result = process_with_multiple_loops([10, 20, 30])
+```
+
+When this file is run, we get the following output showing three separate tables (one for each loop):
+
+```console
+$ python demo.py
+
+iteration    number    sum_so_far    i
+-----------  --------  ------------  ---
+0            N/A       0             0
+1            10        10            0
+2            20        30            0
+3            30        60            0
+
+
+iteration    j    sum_so_far    i
+-----------  ---  ------------  ---
+0            N/A  60            0
+1            0    61            0
+2            1    62            0
+3            2    63            0
+
+
+iteration    sum_so_far    i
+-----------  ------------  ---
+0            63            0
+1            62            1
+2            60            2
+3            57            3
+4            53            4
+5            48            5
+```
+
 ### API
 
 ```{eval-rst}
 .. automethod:: python_ta.debug.AccumulationTable.__init__
 ```
 
-The `AccumulationTable` class has the following instance attributes you can access after the `with` statement.
+The `AccumulationTable` class provides a `loops` instance attribute that becomes available after the `with` statement.
 
 ```{eval-rst}
-.. autoattribute:: python_ta.debug.AccumulationTable.loop_variables
-
-.. autoattribute:: python_ta.debug.AccumulationTable.loop_accumulators
+.. autoattribute:: python_ta.debug.AccumulationTable.loops
```

Comment:
Keep `loop_variables` and `loop_accumulators` defined as read-only Python *properties*, but have them only work when `len(self.loops) == 1`, otherwise you can raise a `ValueError`. This will preserve backwards compatibility for the simple case where there's only one loop.